data/variant-mappings.json

```diff
@@ -1,46 +1,68 @@
 {
   "raiselearning.org": {
     "courses": {
-      "80": "mm",
-      "82": "mm",
-      "83": "mm",
-      "86": "mm",
-      "89": "mm",
-      "91": "mm",
-      "92": "mm",
-      "94": "mm",
-      "97": "mm",
-      "98": "mm",
-      "104": "mm",
-      "108": "mm",
-      "110": "mm",
-      "112": "mm",
-      "117": "mm",
-      "119": "mm",
-      "120": "mm",
-      "123": "mm",
-      "126": "mm",
-      "127": "mm",
-      "128": "mm",
-      "129": "mm",
-      "131": "mm",
-      "133": "mm",
-      "134": "mm",
-      "136": "mm",
-      "182": "mm",
-      "183": "mm",
-      "185": "mm",
-      "188": "mm",
-      "190": "mm",
-      "193": "mm",
-      "194": "mm",
-      "202": "mm",
-      "208": "mm",
-      "210": "mm",
-      "211": "mm",
-      "214": "mm",
-      "230": "mm",
-      "232": "mm"
+      "256": "animate-lomp-variant-a",
```

Comment:
Check my 👀 , but these variant names seem inconsistent with the content repo. It looks like the latter is using `animate-lomp-cameo-a` and `animate-lomp-cameo-b`.